lib/challenge.js

```diff
@@ -51,7 +72,18 @@ var pet_owner;
 var some_pet;
 
 // Answer Starts Here
-
+var some_pet = {
+  name: "Rupert",
+  species: "Dog",
+  breed: "Goldendoodle",
+  noise: "Bark",
+  makeNoise: function makeNoise(){
+    return this.noise;
+  }
+    }
+  }
+  };
+};
```

Comment:
This object is good. It has the needed `noise` attribute and the requested `makeNoise` method. However, there are three too many closing curly braces following it. Every bracket must be matched in correct code. Every unmatched bracket, whether opening or closing, is a syntax error that will cause the interpreter to give up on your code. 9/10